.env

```diff
@@ -1,5 +1,7 @@
 registry=exadel/
+postgres_user=postgres
```

Comment:
In this file the name is `postgres_user` and in `dev/.env` it's `postgres_username`

Reply:
I have corrected them.